Dockerfile

```diff
@@ -1,3 +1,31 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 as builder
+
+# Install build dependencies for the package(s) below
+RUN \
+  yum -y install \
+    autoconf automake bison gettext-devel libtool make pkgconfig tar xz
```

Comment:
This could be simplified as `RUN yum group install -y "Development Tools"`, but if we don't anticipate building more packages any time soon this will avoid 50+ more packages in the builder stage.

Reply:
I think what you have is sufficient for now :+1: